backend/molgenis-emx2-io/src/main/java/org/molgenis/emx2/io/tablestore/TableStoreForURL.java

```diff
@@ -0,0 +1,82 @@
+package org.molgenis.emx2.io.tablestore;
+
+import static org.molgenis.emx2.io.tablestore.TableStoreForCsvFilesDirectory.CSV_EXTENSION;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.StreamSupport;
+import org.molgenis.emx2.BinaryFileWrapper;
+import org.molgenis.emx2.MolgenisException;
+import org.molgenis.emx2.Row;
+import org.molgenis.emx2.io.readers.CsvTableReader;
+
+/** We test this in webapi */
+public class TableStoreForURL implements TableAndFileStore {
+  private URL baseURL;
+
+  public TableStoreForURL(URL baseURL) {
+    if (!baseURL.toString().endsWith("/")) {
+      try {
+        this.baseURL = new URL(baseURL + "/");
+      } catch (Exception e) {
+        // should never happen
```

Comment:
and will never know if it does happen, suggest handeling the error

Reply:
Sure, I will make a useless throw out of it.